File: esercizi_python/compito_python_3/Squash.py
from Campo import Campo


class Squash(Campo):
    def __init__(self, cod, nome, larghezza, lunghezza, costorario, altezza, piano):
        super().__init__(cod, nome, larghezza, lunghezza, costorario)
        self.altezza = altezza
        self.piano = piano

    def toString(self):
        return "squash\t"+super().toString()+"-\t-\t"+str(self.altezza)+"\t"+str(self.piano)+"\t"+str(self.costorario)

File: esercizi_java/NetBeansProjects/Circolo_Sportivo/src/circolo_sportivo/Squash.java
package circolo_sportivo;

/**
 *
 * @author tobiasacchetto
 */
public class Squash extends Campo{
    private int piano;
    private float altezza;
    
    public Squash(int cod,String nome,float larghezza, float lunghezza, float costorario, float altezza, int piano){
        super(cod,nome,larghezza,lunghezza,costorario);
        this.altezza=altezza;
        this.piano=piano;
    }
    
    public String toString(){
        return "squash\t"+super.toString()+"-\t-\t"+altezza+"\t"+piano+"\t"+super.costorario;
    }
    
}
